projects/luxurify.md

```diff
@@ -0,0 +1,85 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
+
+Luxurify
+
+## Category
+<!--developer tooling, application, wallet, infrastructure, etc-->
+Application
+
+## Project Description
+<!--Describe your project in a few sentences. -->
+The Luxurify ecosystem is a platform featuring luxury watch registration, verification, marketplace and an escrow service. The provenance of your watch is inscribed in blockchain.
+
+### Build for Brand
+- Register your brand with certificates
+- Claim watches and start selling.
+- Protection against fakes.
+- Promotion of the official service network.
+- Transparent ownership transfers.
+- Dynamic pricing via WatchSignals.
+
+### Build for Owner
+- Protection against fakes and stolen property.
+- Transparent ownership transfers.
+- Growth in resale value.
+- Ethernal provenance record.
+- Dynamic pricing via WatchSignals.
+- Convenient marketplace.
+
+### Build for Collector
+- Convenient marketplace.
+- Fast, Secure, Trusted place to buy Authentic Luxury Watches.
+
+
+## Project Status
+<!--brainstorming, fundraising, under development, beta, shipped, etc-->
+Under Development
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+https://youtu.be/SMfm2SZ-3rY
+
+## Target Audience
+<!--Describe who will be your project's users-->
+Watch's brands, owners and collectors
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+N/A
+
+## Github repo
```

Comment:
Hi, please add a license file and license as MIT

Reply:
Added MIT license @autonome